src/main/resources/ui_de_DE.properties

```diff
@@ -199,7 +199,7 @@ emailInstructions=Bitte geben Sie hier eine gültige E-Mail-Adresse ein, wenn Si
 finishWizard=Bitte beachten Sie, dass Ihre Identität aktuell nur auf diesem Gerät zur Verfügung steht. \
   Qabel oder Dritte haben keine Kopien auf Ihren Servern. Exportieren Sie zur Sicherheit \
   als Backup Ihre Identität als .qid und bewahren Sie diese sicher auf. \
-  Die Identität mit der Endung .qid enthält den privaten Schlüssel Ihrer Identität, \
+  Die Identität mit der ndung .qid enthält den privaten Schlüssel Ihrer Identität, \
```

Comment:
fixed in https://github.com/Qabel/qabel-desktop/pull/479/commits/53e7708427fe7cee67d56d8569663d81814012e0